client/SymbolClient.py

```diff
@@ -252,7 +254,7 @@ def get_harvests(self, address, start_id=None):
 
             snapshot = TransactionSnapshot(address, 'harvest')
             snapshot.height = int(json_statement['height'])
-            snapshot.timestamp = self._get_block_time_and_multiplier(snapshot.height)[0]
+            (snapshot.timestamp, _, snapshot.hash) = self._get_block_time_and_multiplier_and_hash(snapshot.height)
```

Comment:
at this point wouldn't it be easier to wrap result in @dataclass ? 😬

Reply:
_almost_